### PR TITLE
Add Graphviz installation instructions to installation page.

### PR DESCRIPTION
**Pull request by indhub**
Graphviz is an important component without which graph visualization related MXNet APIs would fail. Adding this to the installation page will reduce the likelihood of users bumping into this issue.

### CODE REVIEW

File: docs/get_started/install.md [hunk 1]
```diff
@@ -111,7 +111,13 @@ Install *MXNet* with OpenBLAS acceleration.
 (mxnet)$ pip install mxnet
 ```
 
-**Step 4**  Validate the installation by running simple *MXNet* code described [here](#validate-mxnet-installation).
+**Step 4**  Install [Graphviz](http://www.graphviz.org/) for graph visualization.
```
Comment (nswamy) [inline docs/get_started/install.md]: While you are on this can you please remove (mxnet) from 105 and 111.

**Comment by piiswrong** [comment]
This shouldn't be put in installation page. Put it in the tutorial index page

**Comment by nswamy** [comment]
@piiswrong this is a dependency of mxnet and should come prepackaged with MXNet. Users commonly use this to visualize the network. 
right now it does not and that is why it is going to the installation page. It does not make sense to just on the tutorials page.

**Comment by piiswrong** [comment]
Then it should be handled by pip, not listed here

**Comment by nswamy** [comment]
I agree. It's already been discussed with @szha in another [PR](https://github.com/dmlc/mxnet/pull/6533) and also offline. The libraries that are used by MXNet should go in PIP/have instruction when built from source instead of being surprised with a stack trace when you use the API.
Right now we want to make sure the user has smooth experience until we add it to PIP

**Comment by indhub** [comment]
@piiswrong ,
1. Until `pip install mxnet` includes graphviz installation, we should atleast inform user through the installation page that graphviz is required.
2. Even after pip installation is modified to include graphviz, 'Build from source' section will still need it.

The PR currently mentions installing graphviz on devices and I want to remove that. Could you please either reopen the PR so I can edit it or explain why the reasons listed above are not good enough to add graphviz installation instructions to 'install' page?

**Pull request closed without merging**